### DISCUSSION TITLE
Should the Network Setup Method Install Elastic?

### DISCUSSION THREAD

**Discussion by kiot389** [Security-Onion-Solutions/securityonion#6888] [Unsupported Versions] [answered]
1. Is Elasticsearch & Kibana installed on Ubuntu 18.04 when you use the network setup? 
2. Should the Elasticsearch & Kibana services be started before running set up?

Summary: 
My install stops at 64% and says check /var/log/elasticsearch. The folder doesn't exist and the services weren't installed. I manually installed them both based on elastic.co's guide, but I get mixed results when I rerun setup? 'sudo service elastic status' says (dead).
The service's unit file lists says notification. Should it be changed or set as simple? I've gotten the install to 96% by messing with the services, but got lucky, because its still stopping at 64%.

**Comment by kiot389** [comment]
Well, I answered my own question pretty quickly. 
1. Elasticsearch, Kibana aren't installed by Security Onion.
2. After Manual install, you should run these commands:

sudo /bin/systemctl daemon-reload
sudo /bin/systemctl enable elasticsearch.service
sudo /bin/systemctl enable kibana.service

**Reply by TOoSmOotH** [reply]
If you select standalone or manager/manager search elastic is installed. You should look at the docs. https://docs.securityonion.net

**Comment by dougburks** [comment, marked as answer]
> 1. Is Elasticsearch & Kibana installed on Ubuntu 18.04 when you use the network setup?

Yes, when you run network setup and choose Import, Eval, Standalone, Manager, or ManagerSearch, then Elasticsearch and Kibana should automatically be installed and configured.

> 2\. Should the Elasticsearch & Kibana services be started before running set up?

No, you shouldn't need to manually install Elasticsearch or Kibana.
